tools/fuzzer/main.c

```diff
@@ -244,7 +245,7 @@ int fuzzer_send_msg(struct fuzz *fuzzer)
 	gettimeofday(&tp, NULL);
 	timeout.tv_sec  = tp.tv_sec;
 	timeout.tv_nsec = tp.tv_usec * 1000;
-	timeout.tv_nsec += 300000000; /* 300ms timeout */
+	timeout.tv_nsec += 1000000000; /* 1000ms timeout */
```

Comment:
When you figure out the magical timeout it's best. Also, why are you putting in a value of 1 billion or more in here, instead of increasing tv_sec instead?

Reply:
Using 1 now.